### PR TITLE
Add `src/directives` folder (vue directives refactor)

### PR DESCRIPTION
**Pull request by Raruto**
Move `src/app/gui/vue/vue.directives.js` file into `src/directives` folder.

**List of changes:**

- create `src/directives` folder
- remove `src/app/gui/vue/directives/v-checked.js`
- remove `src/app/gui/vue/directives/v-disabled.js`
- remove `src/app/gui/vue/directives/v-select2.js`
- ~~deprecate notice for~~ delete `src/app/gui/vue/vue.directives.js`
- consolidate directives definitions within a single folder
- add new file `src/directives/utils.js` (stateful directives helper: `watch` / `unwatch` / `trigger`))

Related to https://github.com/g3w-suite/g3w-client/issues/99

### CODE REVIEW

File: src/directives/index.js [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+      return vm.$watch(() => ApplicationState.lng, () => handlerElement(innerHTML));
+    };
+
+    Vue.directive("disabled", vDisabled);
+    Vue.directive("checked", vChecked);
+    Vue.directive("selected-first", vSelectedFirst);
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t-tooltip.js"):
+     */
```
Comment (Raruto) [inline src/directives/index.js]: @volterra79 does it seem feasible to you?

File: src/directives/index.js [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+        $(el).tooltip('hide');
+        unbindWatch({
+          attr:'g3w-v-t-tooltip-id',
+          el
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t-html.js"):
```
Comment (Raruto) [inline src/directives/index.js]: @volterra79 does it seem feasible to you?

File: src/directives/index.js [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+      unbind(el){
+        unbindWatch({
+          attr:'g3w-v-t-html-id',
+          el
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t-placeholder.js"):
```
Comment (Raruto) [inline src/directives/index.js]: @volterra79 does it seem feasible to you?

File: src/directives/index.js [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+      unbind(el){
+        unbindWatch({
+          attr:'g3w-v-t-placeholder-id',
+          el
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t-title.js"):
```
Comment (Raruto) [inline src/directives/index.js]: @volterra79 does it seem feasible to you?

File: src/directives/index.js [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+      unbind(el){
+        unbindWatch({
+          attr:'g3w-v-t-title-id',
+          el
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t.js"):
```
Comment (Raruto) [inline src/directives/index.js]: @volterra79 does it seem feasible to you?

File: src/directives/index.js [hunk 6]
```diff
@@ -0,0 +1,344 @@
...
+      unbind(el){
+        unbindWatch({
+          el,
+          attr:'g3w-v-t-id'
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-t-plugin.js"):
```
Comment (Raruto) [inline src/directives/index.js]: @volterra79 does it seem feasible to you?

File: src/directives/index.js [hunk 7]
```diff
@@ -0,0 +1,344 @@
...
+      unbind(el){
+        unbindWatch({
+          el,
+          attr: 'g3w-v-t-plugin-id'
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-plugins.js"):
```
Comment (Raruto) [inline src/directives/index.js]: @volterra79 does it seem feasible to you?

File: src/directives/index.js [hunk 8]
```diff
@@ -0,0 +1,344 @@
...
+      unbind(el){
+        unbindWatch({
+          el,
+          attr: 'g3w-v-plugins-id'
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-online.js"):
```
Comment (Raruto) [inline src/directives/index.js]: @volterra79 does it seem feasible to you?

File: src/directives/index.js [hunk 9]
```diff
@@ -0,0 +1,344 @@
...
+      unbind(){
+        unbindWatch({
+          el,
+          attr: 'g3w-v-offline-id'
+        })
+      }
+    });
+
+    /**
+     * TODO: split and refactor into a separated file (eg. "./directives/v-download.js"):
```
Comment (Raruto) [inline src/directives/index.js]: @volterra79 does it seem feasible to you?

File: src/directives/index.js (hunk 1 above)
Comment (Raruto) [reply src/directives/index.js]: Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00

File: src/directives/index.js (hunk 2 above)
Comment (Raruto) [reply src/directives/index.js]: Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00

File: src/directives/index.js (hunk 3 above)
Comment (Raruto) [reply src/directives/index.js]: Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00

File: src/directives/index.js (hunk 4 above)
Comment (Raruto) [reply src/directives/index.js]: Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00

File: src/directives/index.js (hunk 5 above)
Comment (Raruto) [reply src/directives/index.js]: Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00

File: src/directives/index.js (hunk 6 above)
Comment (Raruto) [reply src/directives/index.js]: Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00

File: src/directives/index.js (hunk 7 above)
Comment (Raruto) [reply src/directives/index.js]: Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00

File: src/directives/index.js (hunk 8 above)
Comment (Raruto) [reply src/directives/index.js]: Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00

File: src/directives/index.js (hunk 9 above)
Comment (Raruto) [reply src/directives/index.js]: Closed by https://github.com/g3w-suite/g3w-client/pull/174/commits/316b9af7e433a5e153a58a1343da430f2495a772 and https://github.com/g3w-suite/g3w-client/pull/174/commits/f353c8eda6089d8d1a4e13ab846ab4fe27ee6e00

**Review by volterra79 — APPROVED** [review]

**Pull request merged**